lib/most_frequent_integer.rb

```diff
@@ -3,5 +3,29 @@
 # exists, return nil.
 
 def find_most_frequent_integer(arr)
-  # put your code here
+  
+    #1. Create test array.
+    abs_arr = [1, 1, -1, -1, 2, 3, 4]
```

Comment:
A reminder that, when you use your method in applications but also along with the test file, you should be operating on the input that is passed in to the method as a parameter. This method expects to be passed an Array and gives it a local variable name `arr` on line 5.

Throughout the rest of the method, you do your operations on `abs_arr` which is something I know you were using for testing. This means that in testing, even if I pass the Array `[1,2,3]` to the method, your operations run on this hard-coded `abs_arr`, `[1, 1, -1, -1, 2, 3, 4]`.